docs/metrics/accelerate.md

```diff
@@ -1,4 +1,4 @@
-# Calculated metrics
+# Accelerate metrics
```

Comment:
Like this, the page will have two top-level headings, which is [against the rules](https://www.npmjs.com/package/remark-lint-no-multiple-toplevel-headings). :smile: Instead, we should keep a top-level heading to act as the page title, and then we can have two sections for each group of metrics.

Reply:
Anyway, I really like that we're promoting the "sub-metrics" to its own section on par with the Accelerate metrics, it makes sense to:

* Give them more visibility
* Prepare the structure of the page for subsequent metrics that we might add without incurring in too much nesting

Reply:
> Like this, the page will have two top-level headings, which is [against the rules](https://www.npmjs.com/package/remark-lint-no-multiple-toplevel-headings). 😄 Instead, we should keep a top-level heading to act as the page title, and then we can have two sections for each group of metrics.

Even with the plugin, I wasn't able to understand what you meant. Sorry. 😅 
Let's talk sync.

Reply:
It's fixed now. :raised_hands: Perhaps it was already fixed when I wrote my comment, and I was seeing an outdated version of the file, hence the confusion?